dotnet/src/SemanticKernel.Abstractions/Reliability/DefaultHttpRetryHandlerFactory.cs

```diff
@@ -1,10 +1,12 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 
+using System;
 using System.Net.Http;
 using Microsoft.Extensions.Logging;
 
 namespace Microsoft.SemanticKernel.Reliability;
 
+[Obsolete("Usage of Semantic Kernel internal retry abstractions is deprecated")]
```

Comment:
Can we move this to an extensions package? Same for Polly implementation? `Microsoft.SemanticKernel.Reliability` and/or `Microsoft.SemanticKernel.Reliability.Polly`

Reply:
We can do it in a dedicated PR. Some unit tests using the Polly will need to migrate to there as well.